### DISCUSSION TITLE
Support for ruff in Codacy?

### DISCUSSION THREAD

**Discussion by valeriupredoi** [astral-sh/ruff#10458] [Q&A] [answered]
Hey folks, have you guys tried to get in touch and integrate Ruff into Codacy, in a `codacy-ruff`? Thought I'd first ask here before knocking on Codacy's door. Cheers muchly :beer:

**Comment by MichaReiser** [comment]
That would be neat and would help many to adopt ruff easily. Do you know what's required/involved in adding a linter to Codacy? I did a quick google search but didn't find anything specific on how to go about it.

**Reply by bouweandela** [reply]
It might be best to just contact Codacy and ask. The technical part of it seems straightforward, they have instructions [here](https://github.com/codacy/codacy-engine-scala-seed).

**Reply by valeriupredoi** [reply]
cheers both! I reckon, based on Bouwe's pointer to the integration procedure, that's not so hard (famous last words :grin: ) - do you guys want to start the motions on this? I think there's quite a few folks that would move to ruff this way :beer:

**Reply by valeriupredoi** [reply]
Here, I just started pestering the Codacy guys https://github.com/codacy/codacy-engine-scala-seed/issues/59

**Reply by MichaReiser** [reply]
Thanks. I subscribed to it.

**Reply by MichaReiser** [reply, marked as answer]
Thanks @valeriupredoi for pushing for this on the codacy side. 

There's now official Ruff support with https://github.com/codacy/codacy-ruff

**Reply by valeriupredoi** [reply]
indeed there is, and we've just tested it in our repo - all looks and works good (so far) :beers:
